live/stage/services/bastion/dependencies.tf

```diff
@@ -0,0 +1,19 @@
+data "terraform_remote_state" "vpc" {
+  backend = "remote"
+
+  config = {
+    organization = local.remote_state_organization
+
+    workspaces = {
+      name = local.vpc_remote_state_workspace
+    }
+  }
+}
+
+locals {
+  remote_state_organization  = "debtcollective"
+  vpc_remote_state_workspace = "stage-network"
+  ec2_security_group_id      = data.terraform_remote_state.vpc.outputs.ec2_security_group_id
+  ssh_key_pair_name          = data.terraform_remote_state.vpc.outputs.ssh_key_pair_name
+  subnet_id                  = data.terraform_remote_state.vpc.outputs.public_subnet_ids[0]
```

Comment:
**Question:** Why just the first subnet id? How do we ensure we always have an array of subnet ids and that there's at least one?

Reply:
this is the bastion, we have 2 [public subnets], one for each availability zone. It doesn't matter on which subnet the bastion is placed, but it has to be a public subnet so we can connect to it from the internet.

This is how our vpc looks., except we have two availability zones instead of one

![image](https://user-images.githubusercontent.com/849872/64186247-240d8d80-ce34-11e9-8a7f-207d0b908b39.png)